cmd/natc/natc.go

```diff
@@ -70,6 +71,17 @@ func main() {
 		log.Fatalf("site-id must be in the range [0, 65535]")
 	}
 
+	var ignoreDstTable *bart.Table[bool]
+	if *ignoreDstPfxStr != "" {
```

Comment:
If `*ignoreDstPfxStr == ""` then the range loop will run 0 times, so you can just drop the outer condition and unindent everything by one.